src/pages/ConfirmRegistration.jsx

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from 'react'
+import qs from 'qs';
+
+import api from '../api'
+import styled from 'styled-components'
+
+const Title = styled.h1.attrs({
+    className: 'h1',
+})``
+
+const Wrapper = styled.div.attrs({
+    className: 'form-group',
+})`
+    margin: 30px 80px;
+`
+
+const Button = styled.button.attrs({
+    className: `btn btn-primary`,
+})`
+    margin: 15px 15px 15px 5px;
+`
+
+class ConfirmRegistration extends Component {
```

Comment:
Please use the functional approach to build components with Hooks - https://uk.reactjs.org/docs/hooks-intro.html